CONTRIBUTING.md

```diff
@@ -0,0 +1,116 @@
+# Contributing Guide
+
+If you do not have write permissions, but would like to make a contribution, please refer to the following guidelines.
+
+## Pull Request Guidelines
+
+Before submitting a pull request, please make sure to meet the following:
+
+1. Include tests for bug fixes. Make sure the tests fail if the bug persists and pass if the bug is resolved.
```

Comment:
How about adding an inline template test, at least so that authors could get the calling framework straight?